src/main/scala/com/databricks/spark/csv/CsvParser.scala

```diff
@@ -118,5 +120,24 @@ class CsvParser {
     sqlContext.baseRelationToDataFrame(relation)
   }
 
-}
+  def csvRdd(sqlContext: SQLContext, csvRDD: RDD[String]): DataFrame ={
+
+    val relation: CsvRelation = CsvRelation(
+      () => csvRDD,
+      None,
+      useHeader,
+      delimiter,
+      quote,
+      escape,
+      comment,
+      parseMode,
+      parserLib,
+      ignoreLeadingWhiteSpace,
+      ignoreTrailingWhiteSpace,
+      schema,
+      charset,
```

Comment:
Can be removed after you update ```CsvRelation``` definition. Please also remove form other places where we pass ```charset``` when constructing ```CsvRelation```.